internal/billing/command.go

```diff
@@ -14,11 +14,14 @@ func New(prerunner pcmd.PreRunner) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:         "billing",
 		Short:       "Manage Confluent Cloud billing.",
-		Annotations: map[string]string{pcmd.RunRequirement: pcmd.RequireNonAPIKeyCloudLogin},
+		Annotations: map[string]string{pcmd.RunRequirement: pcmd.RequireCloudLoginAllowFreeTrialEnded},
```

Comment:
I made this change to accommodate the admin subcommands. I added `RequireNonAPIKeyCloudLogin` to the `cost` and `price` subcommands as a result.